From b990b10e3364049b5090e49373f871960f08739b Mon Sep 17 00:00:00 2001
From: Joao Martins <joao.martins@neclab.eu>
Date: Mon, 25 Aug 2014 23:03:38 +0200
Subject: [PATCH] Expose bridge txsync/rxsync functions

---
 sys/dev/netmap/netmap_vale.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/sys/dev/netmap/netmap_vale.c b/sys/dev/netmap/netmap_vale.c
index 2e86443..a1238af 100644
--- a/sys/dev/netmap/netmap_vale.c
+++ b/sys/dev/netmap/netmap_vale.c
@@ -2050,6 +2050,8 @@ netmap_bwrap_attach(struct ifnet *fake, struct ifnet *real)
 	na->nm_register = netmap_bwrap_register;
 	// na->nm_txsync = netmap_bwrap_txsync;
 	// na->nm_rxsync = netmap_bwrap_rxsync;
+	na->nm_txsync = netmap_vp_txsync;
+	na->nm_rxsync = netmap_vp_rxsync;
 	na->nm_config = netmap_bwrap_config;
 	na->nm_krings_create = netmap_bwrap_krings_create;
 	na->nm_krings_delete = netmap_bwrap_krings_delete;
-- 
1.7.10.4

